irc/getters.go

```diff
@@ -374,6 +374,9 @@ func (client *Client) SetRealname(realname string) {
 	client.stateMutex.Lock()
 	client.realname = realname
 	client.stateMutex.Unlock()
+	if client.alwaysOn {
```

Comment:
I missed this earlier: this is technically a data race. (You can `alwaysOn := client.alwaysOn` up above while holding the lock.)